Pulumi.prod.yaml

```diff
@@ -1,5 +1,5 @@
 config:
-  debitor-portal-app:dev-domain: debitor.flexisoft.bjerk.dev.
+  debitor-portal-app:domain: debitor.fpx.no.
```

Comment:
This should be just `fpx.no` right? 🤔

Reply:
I think we should reserve this domain for a website or whatever else. This domain won't be used to access an app in itself